### PR TITLE
fix: wrong data send to create auction

### PR DESCRIPTION
**Pull request by chandni-vault-hill**
## Description

Issue : https://github.com/Vault-Hill/vhc-marketplace-app/issues/111

### CODE REVIEW

File: src/features/partials/SellAsset/index.tsx [hunk 1]
```diff
@@ -65,21 +67,17 @@ const SellAsset = ({ asset }: { asset: AssetWithListing | undefined }) => {
   } = useForm()
 
   const handleAuction = async () => {
```
Comment (jmulq) [inline src/features/partials/SellAsset/index.tsx:69]: These names could be more explanatory. Are we creating an auction? Why not `handleCreateAuction`?

Give context to people working on this in the future, without making something too wordy :D

**Review by jmulq — APPROVED** [review]
Minor comment. Are there no designs for reserve price field on the form?

File: src/features/partials/SellAsset/index.tsx (hunk 1 above)
Comment (chandni-vault-hill) [reply src/features/partials/SellAsset/index.tsx]: I will take care of it next time

**Comment by chandni-vault-hill** [comment]
> Minor comment. Are there no designs for reserve price field on the form?

No, will ask Ozenua

**Review by jmulq — APPROVED** [review]
